### PR TITLE
Submissions backend

### PR DESCRIPTION
**Pull request by willarderic**
**What:**
Added a submissions api call to receive zip files from the user and send them to the arena.

**Why:**
So that the users can upload their code for it to be ran in the arena

### CODE REVIEW

File: server/routers/login.js [hunk 1]
```diff
@@ -47,6 +48,14 @@ router.post(path + '/', (req, res) => {
   }
   const username = body.username
   const password = body.password
+  if (!sanitizer.isValidUsername(username)) {
+    response.message = 'Bad username'
+    return res.status(400).json(response)
+  }
+  //  if (!sanitizer.isValidPassword(password)) {
```
Comment (willarderic) [inline server/routers/login.js]: Whoops I probably want to uncomment this. Got rid of it for testing.

File: server/vars.js [hunk 2]
```diff
@@ -13,5 +13,7 @@ module.exports = {
   PORT: _.defaultTo(process.env.PORT, 3001),
   PASSWORD_SECRET: _.defaultTo(process.env.PASSWORD_SECRET, 'secret'),
   TOKEN_SECRET: _.defaultTo(process.env.TOKEN_SECRET, 'token_secret'),
-  TOKEN_EXPIRE_TIME: _.defaultTo(process.env.TOKEN_EXPIRE_TIME, '1d')
+  TOKEN_EXPIRE_TIME: _.defaultTo(process.env.TOKEN_EXPIRE_TIME, '1d'),
+  MAX_FILE_SIZE: _.defaultTo(process.env.MAX_FILE_SIZE, '50000'),
+  ARENA_SUBMISSIONS_HOST: _.defaultTo(process.env.ARENA_SUBMISSIONS_HOST, 'asdf')
```
Comment (willarderic) [inline server/vars.js]: Might want to put an actual default here...

**Review by michael-beaver — COMMENTED** [review]
Made the pointed out changes as well as fixed a bit of styling

**Review by michael-beaver — APPROVED** [review]
Tested against the arena setup, seems to work as expected.

**Pull request merged**